android/build.gradle

```diff
@@ -30,5 +30,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.15.+'
+    compile 'com.facebook.react:react-native:+'
```

Comment:
I dont think there's much you can do about this at the moment other than lock into the major version.